src/rules/valid-expect.ts

```diff
@@ -0,0 +1,297 @@
+/*
+ * This implementation is ported from from eslint-plugin-jasmine.
+ * MIT license, Tom Vincent.
+ */
+
+import {
+  AST_NODE_TYPES,
+  TSESTree,
+} from '@typescript-eslint/experimental-utils';
+import {
+  ModifierName,
+  createRule,
+  getAccessorValue,
+  isExpectCall,
+  isExpectMember,
+  isSupportedAccessor,
+  parseExpectCall,
+} from './utils';
+
+/**
+ * Async assertions might be called in Promise
+ * methods like `Promise.x(expect1)` or `Promise.x([expect1, expect2])`.
+ * If that's the case, Promise node have to be awaited or returned.
+ *
+ * @Returns CallExpressionNode
```

Comment:
remove? or at least just type the return value rather than have it in the comment

Reply:
Ugh I meant to clean that up, b/c it's not valid anyway (it should be `@return(s)`).